test/e2e/tests/metrics/state-snapshots/errors-before-init-opt-in-background-state.json

```diff
@@ -79,15 +79,7 @@
       "networksMetadata": {
         "networkConfigurationId": { "EIPS": {}, "status": "available" }
       },
-      "providerConfig": {
-        "chainId": "0x539",
-        "nickname": "Localhost 8545",
-        "rpcPrefs": "object",
-        "rpcUrl": "string",
-        "ticker": "ETH",
-        "type": "rpc",
-        "id": "networkConfigurationId"
-      },
+      "providerConfig": "object",
```

Comment:
Does this have to stay here for now because of migrations running in e2e?

Reply:
I think so, yes.  I think same reason discussed in https://github.com/MetaMask/metamask-extension/pull/26150#discussion_r1695431277